src/test/java/Tests/UpdateProfileTests.java

```diff
@@ -0,0 +1,212 @@
+	public  void VerifyUpdatingShareMyPerformanceDetailsWithNoOne()throws Exception{
+		userProfilePage.sharePerfWithNoOne.click();
+		Thread.sleep(500);
+		Assert.assertEquals(userProfilePage.getUpdateMessageforsharingPerformanceWith(), "User Profile updated" );
+		}
+	
+	@Test(priority=14, description = "verify update share performance with My Friends")
+	public  void VerifyUpdatingShareMyPerformanceDetailsWithMyFriends()throws Exception{
+		userProfilePage.sharePerfWithFriends.click();
+		Thread.sleep(500);
+		Assert.assertEquals(userProfilePage.getUpdateMessageforsharingPerformanceWith(), "User Profile updated" );
+		}
+	
+	@Test(priority=15, description = "verify update share performance with All User's")
+	public  void VerifyUpdatingShareMyPerformanceDetailsWithAllUsers()throws Exception{
+		userProfilePage.sharePerfWithAllUser.click();
+		Thread.sleep(500);
+		Assert.assertEquals(userProfilePage.getUpdateMessageforsharingPerformanceWith(), "User Profile updated" );
+		}
+	
+	@Test(priority=16, description = "Verifying Update Password with Incorrect Current Password")
+	public void  verifyingupdatepasswordinvalidcurrentpassword()throws Exception{
+		Thread.sleep(1000);
+		userProfilePage.currentPassword.sendKeys("incorrectcurrentpassword");
+		userProfilePage.newPassword.sendKeys("newpassword");
+		userProfilePage.newConfirmPassword.sendKeys("newpassword");
+		userProfilePage.savePasswordBtn.click();
+		Thread.sleep(1000);
+		Assert.assertEquals(userProfilePage.getIncorrectCurrentPasswordMessage(), "The current password does not match with your existing password" );
+	}
+	
+	@Test(priority=17, description = "Verifying Update password with incorrect New password and Confirm password")
+	public void  VerifyingUpdatingMismtachNewPassword()throws Exception{
+		
+		userProfilePage.clearFields();
+		userProfilePage.currentPassword.sendKeys("password");
+		userProfilePage.newPassword.sendKeys("newpassword");
+		userProfilePage.newConfirmPassword.sendKeys("mismatchpassword");
+		userProfilePage.savePasswordBtn.click();	
+		Assert.assertEquals(userProfilePage.getNewPasswordMismatchMessage(), "New password and Confirm password must be same" );
+	}
+	@Test(priority=18, description = "Verifying Update password with same current password and New password")
+	public void  VerifyingUpdatingPasswordWithSameCurrentandNewPassword()throws Exception{
+		userProfilePage.clearFields();
+		userProfilePage.currentPassword.sendKeys("password");
+		userProfilePage.newPassword.sendKeys("password");
+		userProfilePage.newConfirmPassword.sendKeys("password");
+		Assert.assertEquals(userProfilePage.getSameNewandCurrentPasswordMessage(), "Current password and New password must be different" );
+		
+	}
+	
+	/*@Test(priority=19, description = "Verifying Update password with positive case")
+	public void  VerifyingUpdatingPassword()throws Exception{
+		userProfilePage.clearFields();
+		userProfilePage.currentPassword.sendKeys("password");
```

Comment:
create one common method for updatepassword and use it here

Reply:
created method :- updatePasswordValidCase()